AddressesDataPipeline/serverless.yml

```diff
@@ -5,6 +5,15 @@ provider:
   runtime: dotnetcore3.1
   vpc: ${self:custom.vpc.${opt:stage}}
   stage: ${opt:stage}
+  package:
+    artifact: bin/release/netcoreapp3.1/AddressesDataPipeline.zip
+  environment:
+    DB_HOST: ${ssm:/addresses-api/${self:provider.stage}/postgres-hostname}
```

Comment:
Are new parameter store going to be created for these, or are we linking it to the existing one?

Reply:
These are already in parameter store. Moving them into this section of the serverless file will make them available to both lambda functions in declared here. Before they were inside one of the lambda functions definition